src/security/jwt/security.jwt.guard.ts

```
@@ -0,0 +1,119 @@
+            if (err instanceof UnauthorizedException || err instanceof ForbiddenException) {
+                throw err;
+            }
+            throw new InternalServerErrorException(err.message || 'Unexpected error occurred');
+        }
+
+        return true;
+    }
+
+    private extractTokenFromHeader(request: Request): string | null {
+        const authHeader = request.headers.authorization;
+        if (authHeader && authHeader.startsWith('Bearer ')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)